interactors/nonceHandlerV3/nonceTransactionsHandler.go

```diff
@@ -183,8 +183,10 @@ func (nth *nonceTransactionsHandlerV3) SendTransactions(ctx context.Context, txs
 // Close will cancel all related processes.
 func (nth *nonceTransactionsHandlerV3) Close() {
 	nth.mutHandlers.RLock()
-	defer nth.mutHandlers.RUnlock()
-	for _, handler := range nth.handlers {
+	handlers := nth.handlers
```

Comment:
I do not think this is correct as the slice pointer remains the same. Please use `copy` here